src/shared/components/VideoPlayer.tsx

```diff
@@ -28,7 +24,7 @@ export default class VideoPlayer extends React.Component<VideoPlayerPropsInferfa
     componentDidMount() {
         // eslint-disable-next-line
         videojs.Vhs.xhr.beforeRequest = (options: any) => {
-            options.uri = `${options.uri}${this.options?.token}`
+            options.uri = `${options.uri}`
```

Comment:
Useless assignment ?

Reply:
The video isn't working without this assignment